poetry.lock

```diff
@@ -568,6 +568,20 @@ category = "dev"
 optional = false
 python-versions = "*"
 
+[[package]]
+name = "motor"
+version = "2.5.1"
+description = "Non-blocking MongoDB driver for Tornado or asyncio"
+category = "main"
+optional = false
```

Comment:
Please update lockfile also, this should become `true`
https://github.com/aiogram/aiogram/blob/6e39f9fada99906567d889c9618f65bc2f089c44/poetry.lock#L47